src/gpu/nvidia/cudnn_convolution.cpp

```diff
@@ -33,47 +34,44 @@ status_t cudnn_convolution_fwd_t::execute_convolution(
     return cuda_stream->interop_task([&](::sycl::handler &cgh) {
         using scratch_acc_t = ::sycl::accessor<uint8_t, 1,
                 ::sycl::access::mode::read_write>;
-        auto x_acc = CTX_IN_ACCESSOR(DNNL_ARG_SRC);
-        auto weights_acc = CTX_IN_ACCESSOR(DNNL_ARG_WEIGHTS);
-        auto y_acc = CTX_OUT_ACCESSOR(DNNL_ARG_DST);
-        std::shared_ptr<
-                ::sycl::accessor<uint8_t, 1, ::sycl::access::mode::read>>
-                bias_acc;
+
+        auto arg_src = CTX_IN_SYCL_MEMORY(DNNL_ARG_SRC);
+        auto arg_weights = CTX_IN_SYCL_MEMORY(DNNL_ARG_WEIGHTS);
+        auto arg_dst = CTX_OUT_SYCL_MEMORY(DNNL_ARG_DST);
+
+        sycl_memory_arg<::sycl::access::mode::read> arg_bias;
```

Comment:
We can do `auto arg_bias = CTX_IN_SYCL_MEMORY(DNNL_ARG_BIAS)` unconditionally, arg_bias will be empty.